Makefile

```diff
@@ -39,10 +21,13 @@ example-executor:
 	go build -o ${EXAMPLES}/$@ ${EXAMPLES}/example_executor.go
 
 format:
-	go fmt ${LIBS:%=$(PKG_PREFIX)/%}
+	gofmt -s -w .
+
+formatted:
+	! gofmt -s -d . 2>&1 | read
 
 vet:
-	go vet ${LIBS:%=$(PKG_PREFIX)/%}
+	go vet ./...
 
-test test.v:
-	flags=""; test "$@" != "test.v" || flags="-test.v"; pkg="${TEST}"; test -n "$$pkg" || pkg="${LIBS:%=$(PKG_PREFIX)/%}"; go test $$pkg $$flags
+test: formatted vet
```

Comment:
> Why would we provide such functionality through the Makefile

Looking again I see that this `make test TEST=...` impl doesn't really do what I'd want it to anyway. I really want to avoid having to type `go test -v github.com/mesos/mesos-go/detector/zoo`; I'd much rather type `make test TESTS=detector/zoo`. I don't think the current impl supports that notation though so we can scrap it.

I do like `test.v` because then I can see a breakdown of how long each test case takes. Given that it's pretty easy to run `go test -v ./...` we can get rid of `test.v` (though it does keep my fingers happier).